tq/transfer_queue.go

```diff
@@ -283,23 +283,44 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 	tracerx.Printf("tq: sending batch of size %d", len(batch))
 
 	q.meter.Pause()
-	bRes, err := Batch(q.manifest, q.direction, q.remote, batch.ToTransfers())
-	if err != nil {
-		// If there was an error making the batch API call, mark all of
-		// the objects for retry, and return them along with the error
-		// that was encountered. If any of the objects couldn't be
-		// retried, they will be marked as failed.
-		for _, t := range batch {
-			if q.canRetryObject(t.Oid, err) {
-				q.rc.Increment(t.Oid)
-
-				next = append(next, t)
-			} else {
-				q.wait.Done()
-			}
+	var bRes *BatchResponse
+	if q.manifest.standaloneTransferAgent != "" {
```

Comment:
Indeed this could be moved in the `Batch()` function, but I think it might be better to get rid of that method altogether. All it does is to check for len(objects) which is already checked in `tq.Batch()`, and make a very simple call .. and on top of that, it's used in a single place. 

What other callers do you think there would be?